test/tests/functional/pbs_test_tpp.py

```diff
@@ -251,8 +256,7 @@ def test_client_with_mom(self):
         self.server.client = self.hostB
         self.common_steps(resv=True)
 
-    @requirements(num_moms=2, num_comms=1, num_clients=1,
-                  no_comm_on_server=True)
```

Comment:
why was no_comm_on_server removed?

Reply:
I have removed it as the test is doing that change. i.e to disable comm on server host